examples/consumer-driven-contracts/payment-producer/pom.xml

```diff
@@ -15,14 +15,14 @@
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-dependencies</artifactId>
-            <version>${spring.boot.version}</version>
+            <version>2.2.0.RELEASE</version>
             <type>pom</type>
             <scope>import</scope>
         </dependency>
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-web</artifactId>
-            <version>${spring.boot.version}</version>
+            <version>2.2.0.RELEASE</version>
```

Comment:
since you changed the parent pom this is not needed right ?

Reply:
I reverted the changes i made in parent pom and its using older version of spring boot now.